causalpy/experiments/synthetic_control.py

```diff
@@ -41,8 +41,10 @@ class SyntheticControl(BaseExperiment):
         A pandas dataframe
     :param treatment_time:
         The time when treatment occurred, should be in reference to the data index
-    :param formula:
-        A statistical model formula
+    :param control_units:
+        A list of control units to be used in the experiment
+    :param treated_units:
+        A list of treated units to be used in the experiment
```

Comment:
Ok, sorry. i forgot this was synthetic control. So treated as target makes sense. The idea would be to enable multiple treatment groups? Do you want to enable different "equations" for the different treatment units? Or would it be just the same fixed controls for each potential treatment unit?